src/generators/mandoc/index.mjs

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+import { optionToMandoc, envToMandoc } from './utils/converter.mjs';
+import { writeFile, readFile } from 'node:fs/promises';
+import { join } from 'node:path';
+
+/**
+ * This generator generates a mandoc version of the API docs
+ *
+ * @typedef {Array<ApiDocMetadataEntry>} Input
+ *
+ * @type {import('../types.d.ts').GeneratorMetadata<Input, string>}
+ */
+export default {
+  name: 'mandoc',
+
+  version: '1.0.0',
+
+  description: 'Generates the `node.1` file.',
```

Comment:
May be add what is mandoc a url to the wikipedias page can work. Just to give context of non core contributor

Reply:
I added a URL as a comment to the top of the file, would you like it in the description also?